activestorage/app/javascript/activestorage/ujs.js

```diff
@@ -7,6 +7,10 @@ let started = false
 export function start() {
   if (!started) {
     started = true
+    const inputs = findElements(document, "input[type=submit]")
+    inputs.forEach(function(input) {
+      input.addEventListener("click", tagClickedButton)
+    })
```

Comment:
This won't handle clicks on dynamically inserted elements (ajax loaded content, Turbolinks page change, etc).